db/schema.rb

```diff
@@ -740,7 +740,6 @@
   add_foreign_key "blood_sugars", "facilities"
```

Comment:
All the changes here for `schema.rb` should be removed -- this happens due to database version differences.  You can check out the copy from master like so:

```
  git checkout origin/master -- db/schema.rb
```

Then commit that change to get schema back to the master version.